README.md

```diff
@@ -1135,25 +1135,25 @@ amount of time before you vest at all.
   and companies.
   But they can be abused and their complexity can lead to misunderstandings.
 
-  - The intention of a cliff is to make sure someone is serious about contributing to a
-    company for a significant period before they sign an employment agreement.
+  - The intention of adding cliffs to an employment agreement is to make sure the person
```

Comment:
Oops, may have missed this on last review, but this edit, too, changes sense.

Cliffs officially aren't added to the employment agreement; the cliffs might show up in the offer letter or be equity award (whatever form that takes), typically not the employment agreement.

Revert or rephrase to preserve previous meaning.

Reply:
Fixed. (Reverted completely because this is still the sense I'm getting from the original, so I'm missing something. Do you mean to say that everyone knows about cliffs and everyone expects to see a cliff in their equity agreement, so if they don't want one they won't sign the employment agreement, though those are not the same and you don't necessarily receive them at the same time? Why are we mentioning the employment agreement?)